licenses/Mini-XML

```diff
@@ -1,26 +1,26 @@
 			   Mini-XML License
```

Comment:
Could you also mention that the new license applies to all current versions in the commit message? This will help others understand why the change was done in place.

Reply:
Done.

Reply:
Portage is a package manager, and it doesn't contain Mini-XML.

Reply:
Ok, changed.